### ISSUE TITLE
(Bug) Can not start new crowdsale if  previous one hasn't been finished

### ISSUE THREAD

**Issue by dennis00010011b** [poanetwork/token-wizard#1218]
Steps:
 1. Start creating new crowdsale: minted, 1 tier, any valid parameters
 2. Proceed till Publish page
 3. Close the page(don't close browser)
 4. Open TW's Home page  
 5. Click button 'New crowdsale' 

Expected result : 
 - user should be able to start new crowdsale

Actual result:
 - button 'New crowdsale'  unresponsive
 - user can't start new crowdsale unless finish previous one or clean browser's cookies

If you are reporting a problem with Token Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

*localhost*

### If you were able to create it, what is the URL of your crowdsale?

*n/a*

### Do you have screenshots showing the problem?

*https://www.useloom.com/share/d23582930d3f4ce7a704b0d9bf771453r*

### Do you see errors in the dev console? If yes, please include a screenshot

*No*

---

**Comment by fernandomg** [comment]
@dennis00010011b that's right. But, for instance, if you go directly to StepOne screen (`http://localhost:3000/1`) you'll be able to see the `cancel/continue` screen.

Note: that sceen wasn't redesigned yet. I don't know if it's part of another task or if there's an issue for it. @gabitoesmiapodo can you confirm?

**Comment by gabitoesmiapodo** [comment]
@fernandomg 

There's no issue or design that I remember for that Resume / Cancel section.

We could reuse and expand some designs for that. For example the one at #1178

**Comment by dennis00010011b** [comment]
@fernandomg @gabitoesmiapodo 
`cancel/continue` screen in 'Choose crowdsale' section  now

<img width="1408" alt="screen shot 2018-11-07 at 11 51 27 am" src="https://user-images.githubusercontent.com/14067096/48157248-fdb6aa00-e283-11e8-9ed1-b926cad4de4f.png">

I think user should be able to start new crowdsale from 'Home page' in any case
Or dialog `cancel/continue` should be displayed on Home page in attempt to start new crowdsale if previous one is unfinished

**Comment by gabitoesmiapodo** [comment]
@dennis00010011b 

It seems the user can't do anything until he / her finishes or cancels a pending crowdsale creation.

I think you should at least be able to access the "Choose Crowdsale" section?

The option for a dialog on the home page when clicking the "New Crowdsale" button seems good to me, but we should ask @mariano-aguero or @fernandomg if that's possible.

**Comment by dennis00010011b** [comment]
> I think you should at least be able to access the "Choose Crowdsale" section?
> 
Yes, I am able to cancel pending deployment through "Choose Crowdsale" section

**Comment by gabitoesmiapodo** [comment]
@dennis00010011b 

No, sorry, I mean: In my opinion the user should be able to get to that section ("Choose crowdsale"), even if he / she has an unfinished deployment.

Maybe even have it included in the crowdsales' list, marked as unfinished, or something like that.
